src/main/scala/sbttravisci/TravisCiPlugin.scala

```diff
@@ -3,9 +3,26 @@ package sbttravisci
 import sbt._, Keys._
 
 object TravisCiPlugin extends AutoPlugin {
+  private val DefaultScalaVersion = "2.10.6"    // for compatibility with stock sbt
```

Comment:
I'm still afk so can't explain fully, but I think we should bomb by default. Avoids circular dependencies between the two keys that AI had to avoid in my wartremover patch.